lib/websocket-client-simple/client.rb

```diff
@@ -94,7 +94,11 @@ def close
         def open?
           @handshake.finished? and !@closed
         end
-
+        
+        def close?
```

Comment:
`closed?` is more better to this method name. Do you think about?

Reply:
Certainly that is easier to understand.